flow360/component/flow360_params/flow360_params.py

```diff
@@ -131,6 +131,13 @@ class SubsonicInflow(Boundary):
     totalTemperatureRatio: PositiveFloat
     rampSteps: PositiveInt
 
+class SupersonicInflow(Boundary):
+    """SupersonicInflow boundary"""
+
+    type = pd.Field("SupersonicInflow", const=True)
+    density: PositiveFloat = pd.Field(alias="Density")
+    velocity: Tuple[float, float, float] = pd.Field(alias="Velocity")
+    pressure: PositiveFloat = pd.Field(alias="Pressure")
```

Comment:
1. Are all parameters required, or some of them are optional?
2. Can you write docstring to it following numpy style:
see this for example: https://github.com/flexcompute/Flow360/blob/31c5635ebb711f86acab1f8a3be67321da713140/flow360/component/flow360_params/flow360_params.py#L191

3. Can you add test for this BC, somewhere here for example:
https://github.com/flexcompute/Flow360/blob/31c5635ebb711f86acab1f8a3be67321da713140/tests/test_flow360_params.py#L89
4. Add this BC to BC types list:
https://github.com/flexcompute/Flow360/blob/31c5635ebb711f86acab1f8a3be67321da713140/flow360/component/flow360_params/flow360_params.py#L174

and update docstring for boundaries: https://github.com/flexcompute/Flow360/blob/31c5635ebb711f86acab1f8a3be67321da713140/flow360/component/flow360_params/flow360_params.py#L539

Reply:
All parameters are required